### ISSUE TITLE
InvalidArgumentError, Found Inf or NaN global norm.

### ISSUE THREAD

**Issue by CodeXiaoLingYun** [google-research/bert#472]
NO,I am seeing the same error. I also used the same function(tf.clip_by_global_norm),but I found learning rate and function are not the key reasons. when i generate Vocab,i set the size is 4682,and the vocab_size is 4682 in train.py,too.   as the same,i do not know whether decrease the batch size is useful

In train.py
![image](https://user-images.githubusercontent.com/20829946/53679731-78ac4080-3d0b-11e9-9261-81ec579de97d.png)

the error is
![image](https://user-images.githubusercontent.com/20829946/53679742-aa250c00-3d0b-11e9-860e-f79f44a1aa68.png)

**Comment by CodeXiaoLingYun** [comment]
the model is seq2seq model ！！！！！

**Comment by CodeXiaoLingYun** [comment]
i found a questionable point is GPU:0. I think it may be related with my GPU,so i try to add the code like this image
![image](https://user-images.githubusercontent.com/20829946/53682733-7e1d8100-3d33-11e9-9c0b-0ad7dd314ccc.png)
i do not know whether it is useful,i want to try.
